backend/src/admin/admin.service.ts

```diff
@@ -1,26 +1,21 @@
-import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { Injectable, UnauthorizedException, Inject, forwardRef, ConflictException } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
 import { Admin } from './admin.entity';
-import { AuthenticationService } from '../authentication/authentication.service';
-import { RegisterAdminDto, LoginAdminDto } from './dto';
-import { RegisterJuniorDto } from '../junior/dto';
 import * as content from '../content.json';
+import { RegisterAdminDto } from './dto';
+import { hash } from 'bcrypt';
+import { saltRounds } from '../authentication/authentication.consts';
 
 @Injectable()
 export class AdminService {
 
     constructor(
         @InjectRepository(Admin)
         private readonly adminRepo: Repository<Admin>,
-        private readonly authenticationService: AuthenticationService,
     ) { }
 
-    register = async (registrationData: RegisterAdminDto) => this.authenticationService.registerAdmin(registrationData);
-    login = async (loginData: LoginAdminDto) => this.authenticationService.loginAdmin(loginData);
-    registerJunior = async (registrationData: RegisterJuniorDto) => this.authenticationService.registerJunior(registrationData);
-
-    // TODO:  This will be handed to a guard once a clear workflow is provided for admin login.
+    // This will be handed to a guard once a clear workflow is provided for admin login.
     verifyIsAdmin = async (email: string) => { if (!(await this.getUser(email))) { throw new UnauthorizedException(content.NotAnAdmin); } };
 
     async getUser(email: string): Promise<Admin> {
```

Comment:
Not really part of this PR, but just noticed. In my opinion `getAdmin` would be better name for the function